src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -234,6 +234,13 @@ enum class ConfigType : uint16_t {
    * Payload format: `bool`
    */
   ENABLE_WATCHDOG_TIMER = 300,
+
+  /**
+   * A string for identifying a device.
+   *
+   * Payload format: `char[24]`
```

Comment:
Is this string required to be null terminated? I assume not and that it infers length based on the message size, but probably should document one way or the other

Reply:
We might want to consider increasing this limit, or eliminating altogether since the protocol itself doesn't really care about an upper bound. It might make sense to have this be platform-specific.